src/awkward/operations/convert.py

```diff
@@ -970,10 +970,16 @@ def to_list(array):
     elif isinstance(array, ak.layout.NumpyArray):
         if array.format.upper().startswith("M"):
             return (
-                ak.nplike.of(array)
-                .asarray(array.view_int64)
-                .view(array.format)
-                .tolist()
+                [
```

Comment:
@jpivarski - couldn't figure out why `tolist` returns integers for an array with `M` format...

Reply:
thanks! fixed in https://github.com/scikit-hep/awkward-1.0/pull/911